pkg/codegen/testing/test/testdata/simple-enum-schema/nodejs/types/enums/index.ts

```diff
@@ -25,6 +25,7 @@ export const CloudAuditOptionsLogName = {
      * What if triple quotes """ are used in the description
      */
     Synthetic: "SYNTHETIC",
+    CloudAuditOptionsLogName_NO_NAME: "_NO_NAME",
```

Comment:
Why do we need the whole enum type name here? Identifiers are ok to start with underscore.

Reply:
I don't know. My suspicion is that it provides a way to guarantee uniqueness for when enums don't have unique namespaces. I don't know why this feature is present in nodejs. I'll make a note to clean this up for 4.0.